docs/guides/miscellaneous/how_to_use_the_great_expectations_cli.md

```diff
@@ -2,4 +2,96 @@
 title: How to use the Great Expectations command line interface (CLI)
 ---
 
-This article is a stub.
+The Great Expectations command line is organized using a **`<NOUN> <VERB>`** syntax.
+This guide is organized by nouns (datasource, suite, docs) then verbs (new, list, edit, etc).
+
+## Basics
+
+You can get a list of Great Expectations commands available to you by typing `great_expectations --v3-api --help`.
+Each noun command and each verb sub-command has a description, and should help you find the thing you need.
+
+Please note that the V3 (Batch Request) API can be accessed by adding the `--v3-api` flag after `great_expectations`. Please see our how-to guides and use the `--help` flag for more complete descriptions of the new functionality.
+
+:::note All Great Expectations commands have help text which you can access by adding `--help` to the end of a command. For example, by running `great_expectations --v3-api suite new --help` you'll see help output for that specific command.
+:::
```

Comment:
![Screen Shot 2021-07-22 at 9 17 24 AM](https://user-images.githubusercontent.com/49923762/126645253-0afa2a68-1624-4df6-b66f-7110be1776fc.png)
```suggestion
:::info [OPTIONAL_TEXT_HERE]
All Great Expectations commands have help text which you can access by adding `--help` to the end of a command. For example, by running `great_expectations --v3-api suite new --help` you'll see help output for that specific command.
:::
```
I think an `info` box would be a bit more appropriate here. You can change the specific title or omit to have `INFO` be the header.